Makefile

```diff
@@ -82,33 +83,33 @@ restart_girder:
                                         | jq -r .authToken.token)"
 
 	
-rebuild_dashboard: src/dashboard
+rebuild_dashboard_old: src/dashboard
 	sed -e "s|apiHOST|https://girder.local.wholetale.org|g" \
 		-e "s|dashboardHOST|https://dashboard.local.wholetale.org|g" \
 		-e "s|dataOneHOST|https://cn-stage-2.test.dataone.org|g" \
 		-e "s|dashboardDev|true|g" \
 		-e "s|authPROVIDER|Globus|g" -i src/dashboard/config/environment.js
 	docker run --rm -ti -v $${PWD}/src/dashboard:/usr/src/node-app -w /usr/src/node-app node:carbon-slim sh -c 'NODE_ENV=development npm install && ./node_modules/.bin/ember build --environment=production'
 
-watch_dashboard: src/dashboard
+watch_dashboard_old: src/dashboard
 	sed -e "s|apiHOST|https://girder.local.wholetale.org|g" \
                 -e "s|dashboardHOST|https://dashboard.local.wholetale.org|g" \
                 -e "s|dataOneHOST|https://cn-stage-2.test.dataone.org|g" \
 		-e "s|dashboardDev|true|g" \
                 -e "s|authPROVIDER|Globus|g" -i src/dashboard/config/environment.js
 	docker run --rm -ti -v $${PWD}/src/dashboard:/usr/src/node-app -w /usr/src/node-app node:carbon-slim sh -c 'NODE_ENV=development npm install && ./node_modules/.bin/ember serve --environment=production'
 
-watch_dashboard_dev: src/dashboard
+watch_dashboard_old_dev: src/dashboard
 	sed -e "s|apiHOST|https://girder.local.wholetale.org|g" \
                 -e "s|dashboardHOST|https://dashboard.local.wholetale.org|g" \
                 -e "s|dataOneHOST|https://cn-stage-2.test.dataone.org|g" \
                 -e "s|authPROVIDER|Globus|g" -i src/dashboard/config/environment.js
 	docker run --rm -ti -v $${PWD}/src/dashboard:/usr/src/node-app -w /usr/src/node-app node:carbon-slim sh -c 'NODE_ENV=development npm install && ./node_modules/.bin/ember serve'
 
-rebuild_dashboard_next:
+rebuild_dashboard:
 	docker run --rm --user=$${UID}:$${GID} -ti -v $${PWD}/src/ngx-dashboard:/srv/app -w /srv/app bodom0015/ng '${YARN} install --network-timeout=360000 && ${NG} build --prod --deleteOutputPath=false --progress'
```

Comment:
Should bodom0015/ng be pointing to a Whole Tale resource?

Reply:
It should be changed to `wholetale/ngx-dashboard`

Reply:
Okay thanks for confirming. I had made that change at one point and ran across the error pasted below.

```
thomas@thomas-girder:~/deploy-dev$ make rebuild_dashboard
docker run --rm --user=${UID}:${GID} -ti -v ${PWD}/src/ngx-dashboard:/srv/app -w /srv/app wholetale/ngx-dashboard 'node --max_old_space_size=2048 /usr/local/bin/yarn install --network-timeout=360000 && node --max_old_space_size=2048 ./node_modules/@angular/cli/bin/ng build --prod --deleteOutputPath=false --progress'
docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"node --max_old_space_size=2048 /usr/local/bin/yarn install --network-timeout=360000 && node --max_old_space_size=2048 ./node_modules/@angular/cli/bin/ng build --prod --deleteOutputPath=false --progress\": stat node --max_old_space_size=2048 /usr/local/bin/yarn install --network-timeout=360000 && node --max_old_space_size=2048 ./node_modules/@angular/cli/bin/ng build --prod --deleteOutputPath=false --progress: no such file or directory": unknown.
Makefile:110: recipe for target 'rebuild_dashboard' failed
make: *** [rebuild_dashboard] Error 127
```

I went ahead and tried to check that yarn was in fact installed in `/usr/local/bin/yarn` but was having an issue getting an instance of bash.

```
thomas@thomas-girder:~/deploy-dev$ docker exec -it 95829a0735e2 /bin/bash
OCI runtime exec failed: exec failed: container_linux.go:349: starting container process caused "exec: \"/bin/bash\": stat /bin/bash: no such file or directory": unknown
thomas@thomas-girder:~/deploy-dev$ docker exec -it 95829a0735e2 bash
OCI runtime exec failed: exec failed: container_linux.go:349: starting container process caused "exec: \"bash\": executable file not found in $PATH": unknown
```


Can either of you confirm or deny similar behaviour?

Reply:
My bad, I haven't looked closely enough. It's using staged build so the end container is a very slim nginx enviroment (no bash, only /bin/sh). It can't be used to build the entire dashboard. Let's keep `bodom0015/ng`

Reply:
@Xarthisius is correct - `wholetale/ngx-dashboard` only contains the built code and NGINX, `bodom0015/ng` is an official `node:carbon` containing the necessary build tools (this may not include testing tools yet, but I can update it to include them if it might help) 

Sorry for the confusion. We can persist that one to a more official resource as well if necessary.